crates/matrix-sdk-crypto/src/identities/device.rs

```diff
@@ -939,7 +939,7 @@ impl ReadOnlyDevice {
     }
 
     /// Get the local timestamp of when this device was first persisted, in
-    /// seconds since epoch.
+    /// seconds since epoch (client local time).
```

Comment:
This value is persisted, right? Since local timezone can be changed, I think we should store things in UTC.

Reply:
… and thus, I repeat my suggestion to use `SecondsSinceUnixEpoch` from ruma, and only convert to `u64` for FFI use. Maybe I wasn't very clear about this, when also touching on all the other possibilities in the Matrix discussion.

Reply:
Not sure to get it, there is a SecondsSinceUnixEpoch in ruma?
I tried not using u64 in the structs, but it leads to several sort of seriaization problems. What's the reason to not use u64 directly

Reply:
Yes: https://docs.ruma.io/ruma/struct.SecondsSinceUnixEpoch.html
It has convenient conversions to / from `SystemTime`, has a `::now()` constructor and an obvious representation (`u64` could be local time / UTC, seconds / milliseconds and so on).

Reply:
I have done that, using it now.

Reply:
You used the milliseconds one, not the seconds one. But I don't personally care either way. The milliseconds one is much more common so maybe it's the better choice 🤷🏼

Reply:
Yes I was converting in millis on the kotlin side, so I just used it directly